src/pages/TermsOfService.js

```diff
@@ -30,28 +33,39 @@ You agree to access and use Terra for lawful purposes only. You are solely respo
 use of Terra.
 
-#### By accessing and using Terra, you agree that you will:
+#### By accessing and using Terra, you agree that you must:
 
 * Conduct only authorized business on the system.
-* Notify the Broad Institute if you believe you are being granted access that you should not have.
 * Access Terra using only your own individual account. Group or shared accounts are NOT permitted.
-* Maintain the confidentiality of your authentication credentials such as your password; a Broad
-  Institute employee should never ask you to reveal your password.
-* Report any security incidents relating to lost passwords to the Broad Institute.
-* Follow proper logon/logoff procedures. You must manually logon to your session and promptly close
-  your browser when your session is no longer active.
+* Maintain the confidentiality of your authentication credentials such as your password. 
+  Do not reveal your authentication credentials to anyone; a Broad Institute employee should never 
+  ask you to reveal them.
+* Report all security incidents or suspected incidents (e.g., lost passwords, improper or 
+  suspicious acts) related to Broad Institute systems and networks to the Broad Institute Operations 
+  Center [infosec@broadinstitute.org](mailto:infosec@broadinstitute.org).
+* Follow proper logon/logoff procedures. You must manually logon to your session; do not 
+  store you password locally on your system or utilize any automated logon capabilities. You must 
+  promptly logoff when session access is no longer needed. If a logoff function is unavailable, you 
+  must close your browser. Never leave your computer unattended while logged into the system.
 * Ensure that Web browsers use Secure Socket Layer (SSL) version 3.0 (or higher) and Transport
   Layer Security (TLS) 1.0 (or higher). SSL and TLS must use a minimum of 256-bit, encryption.
+* Safeguard system resources against waste, loss, abuse, unauthorized use or disclosure, and 
+  misappropriation.
+* Contact the Broad Institute Chief Information Security Officer or the Broad Institute 
+  Operations Center ([infosec@broadinstitute.org](mailto:infosec@broadinstitute.org)) if you do not 
+  understand any of these rules.
 
-#### By accessing and using Terra, you agree that you will NOT:
+#### By accessing and using Terra, you agree that you must NOT:
 
 * Use Terra to commit a criminal offense, or to encourage others to conduct acts that would
   constitute a criminal offense or give rise to civil liability.
-* Browse, search or reveal any protected data except in accordance with that which is required to
-  perform your legitimate tasks or assigned duties.
+* Process U.S. classified national security information on the system.
+* Browse, search or reveal any protected data by Broad Institute except in accordance with that 
+  which is required to perform your legitimate tasks or assigned duties.
 * Retrieve protected data, or in any other way disclose information, for someone who does not have
```

Comment:
The wording Bernick provided said "information" instead of "protected data". I don't know if that is a significant difference. To speculate, "protected data" could be interpreted as "protected health data" while "information" could be interpreted more broadly. While I personally like the way "protected data" reinforces the spirit of the condition, "information" might still be better.

I'm just pointing it out because I see it. I do not presume to have the authority to make the final decision. I also do not know where "protected data" came from in the first place so maybe there are reasons to prefer it that I'm not aware of.

Reply:
I agree with your assessment, it's a tough call. I think I'll add both "information" and "protected data" to cover our bases.